index.d.ts

```diff
@@ -1,8 +1,44 @@
-export function format(input: object, options?: {
-    abbreviate: boolean;
-    appendCountry: boolean;
-    cleanupPostcode: boolean;
-    countryCode: string;
-    fallbackCountryCode: string;
-    output: 'string' | 'array';
-}): string | string[];
+// TODO: add aliases
+/** @see [here](https://github.com/OpenCageData/address-formatting/blob/master/conf/components.yaml) */
+type Input = Partial<
+  Record<
+    | 'houseNumber'
```

Comment:
I added this tiny script that should help with resolving the TODO: https://github.com/fragaria/address-formatter/commit/7c8b694759e14efa6570afeb60310ce112029a7b

Reply:
@JirkaChadima, just to clarify - do you want to handle TODO in this PR?

Reply:
yes, please. We can make it right the first time, I don't think there are tons of changes in the aliases

Reply:
Done, could you check please?

Reply:
Looks good to me! Thank you!